src/components/LoginComponent.astro

```diff
@@ -0,0 +1,76 @@
+---
+
+
+---
+<script>
+  import { inMemoryPersistence } from "firebase/auth";
+  import firebase from "firebase/compat/app";
+  import * as firebaseui from "firebaseui";
+  import "firebaseui/dist/firebaseui.css";
+  import { auth } from "../firebase/client";
+
+  const uiConfig = {
+    signInOptions: [
+      // Leave the lines as is for the providers you want to offer your users.
+      // google, apple, email
+      firebase.auth.GoogleAuthProvider.PROVIDER_ID,
+      // in order to make the switch from signin to signup for email 
+      // remember to disable the the email enumeration protection
+      // https://github.com/firebase/firebaseui-web/issues/1040
+      firebase.auth.EmailAuthProvider.PROVIDER_ID,
+      'apple.com'
+    ],
+    // tosUrl and privacyPolicyUrl accept either url string or a callback
+    // function.
+    // Terms of service url/callback.
+    tosUrl: "<your-tos-url>",
```

Comment:
remove the comments and add a tos url. I would add a very simple page, for now, with just a "Terms of service". we will add an issue to work on it